packages/php-storage-driver-common/proto/table.proto

```diff
@@ -64,6 +64,13 @@ message AddColumnCommand {
   TableColumnShared columnDefinition = 3; // table column definition
 }
 
+message AlterColumnCommand {
+  repeated string path = 1; // path where table is located
+  string tableName = 2; // table name
+  TableColumnShared columnDefinition = 3; // table column definition
```

Comment:
Tohle bude jakože ta "desired"?

Reply:
jo, mam to prejmenovat? jeste uvazuju nad tim, ze potrebuju exsiting definition. puvodne jsem si rikal, ze si to descrbnu, ale na snflk to delas z metadat -> jdu tam jeste pridat "existing"

Reply:
No, pokud máš desired kompletní + toUpdate, tak existing nepotřebuješ. Jen ověříš, že to ta tabulka už má, jak hcceš, ne?

Reply:
A ten desired postavíš už v connection z existing + updated. Ale možná je to anipattern. Pak bys neměl posílat desired komplet asi 🤷

Reply:
Ad přejmenování - no, není to zjevné, že je to desired a že je v tom i name, ale možná to nevadí...

Reply:
no `TableColumnShared` odpovida instanci naseho [datatypu](https://github.com/keboola/storage-backend/blob/main/packages/php-datatypes/src/Definition/Bigquery.php), takze takto je to na pytel - dostavam sem jen "desired", ale chybi mi existing. ktery bych sice mohl udelat nejakou describe operaci, chceme, aby to vychazelo z connection metadat -> jdu to pridat

Reply:
Mě přijde, že driver by to neměl řešit, ne? Prostě mu řekneme "cíl je aby to vypadalo takhle" a on to udělá protože má přístup k tomu zdroji. Proč myslíš, že by to mělo vycházet z metadat?